.github/workflows/test.yml

```diff
@@ -37,15 +37,15 @@ jobs:
           name: barretenberg
 
       - name: Restore nix store cache
-        id: nix-store-cache
-        uses: actions/cache@v3
+        uses: actions/cache/restore@v3
+        id: restore-cache
         with:
           path: /tmp/nix-cache
```

Comment:
```suggestion
          path: ${{ env.CACHED_PATHS }}
```

Reply:
I think this reads better using the literal as it's a single directory and we're using it in other steps as well.

Reply:
Thoughts on making it an env var named something else so we can interpolate it into the other steps as well? There's a lot of duplication to keep in sync

Reply:
Ok, done.